src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -1,13 +1,37 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Truck
- * Do not remove no-field constructor
- */
 public class Truck extends Machine {
+    private String liftingCapacity;
+    private String power;
```

Comment:
The field 'power' should have an access modifier to encapsulate the field properly.